### ISSUE TITLE
ModuleNotFoundError

### ISSUE THREAD

**Issue by MOONCHAND** [PayelGanguly96/Loan-Prediction#2]
I'm having an error called ModuleNotFoundError: No module named 'sklearn.cross_validation'. What can i do, and already installed scikit learn please help me as soon as possible.
![Screenshot (126)](https://user-images.githubusercontent.com/39911581/57752648-7ef56a80-7707-11e9-8583-19030c97e1f2.png)

**Comment by MOONCHAND** [comment]
And when i use model_selection instead cross_validation then also get an Error called
TypeError: __init__() got an unexpected keyword argument 'n_folds' what can i do help me.

**Comment by Shrirang97** [comment]
Have you search first issue on stackoverflow?

I dont know may be you are using old sklearn version. 

Try to update it.

**Comment by MOONCHAND** [comment]
Yes i have searched on stackoverflow.And it says the module  cross_validation is deprecated so instead cross_validation you should use model_selection from now on.
And my Scikit learn version is scikit-learn              0.20.3           py36h343c172_0
As you can see below.
![Screenshot (127)](https://user-images.githubusercontent.com/39911581/57761558-642cf100-771b-11e9-9878-3ed594aca6be.png)

**Comment by MOONCHAND** [comment]
Now it is working ,just needed some changes and worked very well.
![Screenshot (128)](https://user-images.githubusercontent.com/39911581/57766481-06050b80-7725-11e9-9713-df5df67b6449.png)

**Comment by MOONCHAND** [comment]
But I have a problem, don't understand some codes of this cell given below.
Please help me out bro!
![Screenshot (129)](https://user-images.githubusercontent.com/39911581/57766809-ac511100-7725-11e9-808f-24d3821df909.png)

**Comment by Shrirang97** [comment]
which part ?

**Comment by MOONCHAND** [comment]
All of these.
model.fit(data[predictors],data[outcome]),
train_target = (data[predictors].iloc[train,:],
error.append(model.score(data[predictors].iloc[test,:], data[outcome].iloc[test])),
and
 accuracy = metrics.accuracy_score(predictions, data[outcome])
